plugins/com.google.cloud.tools.eclipse.appengine.newproject/src/com/google/cloud/tools/eclipse/appengine/newproject/messages.properties

```diff
@@ -6,6 +6,7 @@ create.app.engine.standard.project=Create a new Eclipse project for App Engine s
 project.creation.failed=Failed to create project
 java.package=Java package:
 illegal.package.name=Illegal package name: {0}
+illegal.service.name=A service name may contain up to 63 numbers, letters, and hyphens, with no leading or trailing hyphens.
```

Comment:
The `{0}` in the `illegal.package.name` is used to show the validation error message produced by our `JavaPackageValidator`; the `ServiceNameValidator` just returns a true/false.